pycontrails/models/cocip/output_formats.py

```diff
@@ -0,0 +1,2003 @@
+    }
+    ds["natural_cirrus"].attrs = {
+        "units": " ", "long_name": "Natural cirrus cover."
+    }
+    ds["contrails"].attrs = {
+        "units": " ", "long_name": "Contrail cirrus cover without overlap with natural cirrus."
+    }
+    return MetDataset(ds)
+
+
+def optical_depth_to_cirrus_coverage(
+        optical_depth: MetDataArray, *,
+        threshold: float = 0.1,
+) -> MetDataArray:
+    """
+    Calculate contrail or natural cirrus coverage in a longitude-latitude grid.
+
+    A grid cell is assumed to be covered by cirrus if the optical depth is above `threshold`
+
+    Parameters
+    ----------
+    optical_depth : MetDataArray
+        Contrail or natural cirrus optical depth in a longitude-latitude grid
+    threshold : float
+        Sensitivity of cirrus detection, set at 0.1 to match the capability of satellites.
+
+    Returns
+    -------
+    MetDataArray
+        Contrail or natural cirrus coverage in a longitude-latitude grid
+    """
+    cirrus_cover = xr.where(optical_depth.data > threshold, 1, 0)
+    return MetDataArray(cirrus_cover)
+
+
+def regional_statistics(da_var: xr.DataArray, *, agg: str) -> pd.Series:
+    """
+    Calculate regional statistics from longitude-latitude grid.
+
+    Parameters
+    ----------
+    da_var : xr.DataArray
+        Air traffic or contrail variable in a longitude-latitude grid.
+    agg : str
+        Function selected for aggregation, (i.e., "sum" and "mean").
+
+    Returns
+    -------
+    pd.Series
+        Regional statistics
+
+    Notes
+    -----
+    - The spatial bounding box for each region is defined in Teoh et al. (2023)
+    - Teoh, R., Engberg, Z., Shapiro, M., Dray, L., and Stettler, M.: A high-resolution Global
+        Aviation emissions Inventory based on ADS-B (GAIA) for 2019–2021, EGUsphere [preprint],
+        https://doi.org/10.5194/egusphere-2023-724, 2023.
+    """
+    if (agg == "mean") & (len(da_var.time) > 1):
+        da_var = da_var.mean(dim=["time"])
+        da_var = da_var.fillna(0)
+
+    # Get regional domain
+    world = da_var.copy()
+    usa = da_var.sel(longitude=slice(-126, -66), latitude=slice(23, 50))
+    europe = da_var.sel(longitude=slice(-12, 20), latitude=slice(35, 60))
+    east_asia = da_var.sel(longitude=slice(103, 150), latitude=slice(15, 48))
+    sea = da_var.sel(longitude=slice(87.5, 130), latitude=slice(-10, 20))
+    latin_america = da_var.sel(longitude=slice(-85, -35), latitude=slice(-60, 15))
+    africa = da_var.sel(longitude=slice(-20, 50), latitude=slice(-35, 40))
+    china = da_var.sel(longitude=slice(73.5, 135), latitude=slice(18, 53.5))
+    india = da_var.sel(longitude=slice(68, 97.5), latitude=slice(8, 35.5))
+    n_atlantic = da_var.sel(longitude=slice(-70, -5), latitude=slice(40, 63))
+    n_pacific_1 = da_var.sel(longitude=slice(-180, -140), latitude=slice(35, 65))
+    n_pacific_2 = da_var.sel(longitude=slice(120, 180), latitude=slice(35, 65))
+    arctic = da_var.sel(latitude=slice(66.5, 90))
```

Comment:
All of these slice values should be float.

Reply:
Perhaps this should be it's own helper function ... you do an identical slicing below into regions.